generate.py

```diff
@@ -13,34 +13,38 @@ async def generate_prompt_response(message, character, context):
     chat_history = chatHistory.ChatHistory(message, character.name).load(character, message.author.display_name)
     user_settings = settings.load_user_settings(message.author, character.name)
     prompt = (context + 
-              "\n" + chat_history[-MAX_CHAT_HISTORY_LENGTH:] + 
+              "\n" + chat_history[-MAX_CHAT_HISTORY_LENGTH:] + # Limit chat history
               "\n" + message.author.display_name + ": " + message.content +
               "\n" + character.name + ":" + user_settings["prefix"] + " ").lstrip()
     print(prompt)
-    response = requests.post(
-        API_ENDPOINT,
-        headers={"Content-Type": "application/json"},
-        json={
-            "prompt": prompt,
-            "max_new_tokens": user_settings["max_response_length"],
-            "min_length": user_settings["min_length"],
-            "temperature": user_settings["temperature"],
-            "repetition_penalty": user_settings["repetition_penalty"],
-            "stopping_strings": [f"{message.author.display_name}:"],
-            "add_bos_token": False,
-            "max_context_length": 8192,
-        }
-    )
+    
+    headers = {
+        "Content-Type": "application/json",
+        # Add your authorization header here if needed
+    }
+    
+    # Prepare the data for the request
+    data = {
+        "messages": [
+            {"role": "system", "content": context},
+            {"role": "user", "content": prompt}
+        ],
+        "max_tokens": user_settings["max_response_length"],
+        "temperature": user_settings["temperature"],
+        "min_tokens": user_settings["min_length"],

```

Comment:
Makes sense, but where is "min_tokens" in the documentation? Doesn't matter I suppose, it works in testing ^^